server/osc_dash.py

```
@@ -55,8 +76,157 @@ def api():
     summary = load_summary()
     wins = load_windows(200)
     live = load_live_snaps()
+    # full rows for goal aggregation (all windows, not 200)
+    all_rows=[]
+    wf = RUN / "oscillation_windows.jsonl"
+    if wf.exists():
+        for line in wf.read_text(encoding="utf-8").splitlines():
+            if not line.strip(): continue
```

Ruff (0.16.2)

[error] Multiple statements on one line (colon)

(E701)